src/Symfony/Component/ExpressionLanguage/Node/BinaryNode.php

```diff
@@ -57,6 +55,28 @@ public function compile(Compiler $compiler)
             return;
         }
 
+        switch ($operator) {
+            case 'not in':
+                $compiler->raw('!');
+                // no break
+            case 'in':
+                $compiler
+                    ->raw('in_array(')
+                        ->compile($this->nodes['left'])
+                        ->raw(', ')
+                        ->raw('is_array(')
+                            ->compile($this->nodes['right'])
+                        ->raw(') ? ')
```

Comment:
I wish I could use ConditionalNode instead of ->raw() for ternary, but this ternary requires FunctionNode inside, for which Parser would have to have functions `is_array` and `iterator_to_array` registered